R/summary_functions.R

```diff
@@ -190,15 +195,16 @@ print.coverages <- function(x, ...) {
 }
 
 format_percentage <- function(x) {
-  color <- if (x >= 90) crayon::green
-    else if (x >= 75) crayon::yellow
-    else crayon::red
+  color <- if (x >= 90) cli::col_green
+    else if (x >= 75) cli::col_yellow
+    else cli::col_red
 
   color(sprintf("%02.2f%%", x))
 }
 
 markers <- function(x, ...) UseMethod("markers")
 
+#' @export
```

Comment:
Fix warnings flagged by roxygen2 7.3.0 about the need to add an `@export` tag to un-exported S3 methods